pkgs/test/cc-wrapper/default.nix

```diff
@@ -131,8 +131,8 @@ in stdenv.mkDerivation {
       ASAN_OPTIONS=use_sigaltstack=0 ${emulator} ./sanitizers
     ''}
 
-    echo "Check whether CC with NIX_CC_USE_RESPONSE_FILE hardcodes all required binaries..." >&2
-    NIX_CC_USE_RESPONSE_FILE=1 ${CC} -v
+    echo "Check whether CC and LD with NIX_X_USE_RESPONSE_FILE hardcodes all required binaries..." >&2
+    NIX_CC_USE_RESPONSE_FILE=1 NIX_LD_USE_RESPONSE_FILE=1 ${CC} -v
```

Comment:
Made the test test this

Reply:
SGTM. I’m currently bootstrapping another `stdenv` so I won’t be able to test this myself tonight. What I’m curious about is why it only broken on Darwin before.

Reply:
> SGTM. I’m currently bootstrapping another `stdenv` so I won’t be able to test this myself tonight. What I’m curious about is why it only broken on Darwin before.

Because clang has `use_response_file_by_default` and the cc-wrapper test is now run without anything in the path.

Reply:
Oh, I see, LLVM and cctools set it where GCC and binutils don’t.